plugins/plugin-site/src/components/InstallInstructions.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Modal, ModalHeader, ModalBody} from 'reactstrap';
+import InstallViaCLI from './InstallViaCLI';
+
+function InstallInstructions({isShowInstructions, toggleShowInstructions, pluginId, pluginVersion}) {
+    return (
+        <Modal placement="bottom" isOpen={isShowInstructions} target="pluginDependencies" toggle={toggleShowInstructions}>
+            <ModalHeader toggle={toggleShowInstructions}>Installation options</ModalHeader >
+            <ModalBody>
+                <ol>
+                    <li>
+                        <a href="https://www.jenkins.io/doc/book/managing/plugins/#from-the-web-ui">Using the GUI:</a>
+                        {' From your Jenkins dashboard navigate to '}
+                        <strong>Manage Jenkins &gt; Manage Plugins</strong>
+                        {' and select the '}
+                        <strong>Available</strong>
+                        {' tab. Locate this plugin by searching for '}
+                        <em>{pluginId}</em>
```

Comment:
this would def be something that my "my.jenkins.io" project I never finished would be good for.